apps/Gruntfile.js

```diff
@@ -127,11 +127,20 @@ module.exports = function (grunt) {
           //TODO: Would be preferrable to separate Blockly media.
           dest: 'build/package/media'
         },
+        // We have to do some weird stuff to get our fallback video player working.
+        // video.js expects some of its own files to be served by the application, so
+        // we include them in our build and access them via static (non-fingerprinted)
+        // root-relative paths.
+        // We may have to do something similar with ace editor later, but generally
+        // we'd prefer to avoid this way of doing things.
+        // TODO: At some point, we may want to better rationalize the package
+        // structure for all of our different assets (including vendor assets,
+        // blockly media, etc).
```

Comment:
it would be 💯 to use webpack's url-loader for everything. But we should definitely punt on this idea for the moment.